### DISCUSSION TITLE
Lighthouse TBT framework.js

### DISCUSSION THREAD

**Discussion by dottodot** [vercel/next.js#47924] [Help]
### Summary

When running my site through lighthouse I'm getting a poor Total Blocking Time of around 2s with the culprit being framework.js

Something I have noticed is on the next.js site the framework.js is very small and only has react.production.min.js included whereas mine has 
react-dom.production.min.js
react-jsx-runtime.production.min.js
react.production.min.js
scheduler.production.min.js

All the pages are built using ISR. Just wondering if I missing something or is framework.js  dependant on how the app is built?

**Comment by icyJoseph** [comment]
Could you share the Lighthouse tree view map link?

**Reply by dottodot** [reply]
I actually found the issue it was a 3rd party animation library causing the issue

**Reply by guymoy1** [reply]
I also have this issue... can you tell which library caused this issue for you?
and how you fixed it?
